cmd/vendor_update.go

```diff
@@ -37,12 +37,22 @@ kubebuilder update vendor
 `,
 		Run: func(cmd *cobra.Command, args []string) {
 			dieIfNoProject()
-			err := (&scaffold.Scaffold{}).Execute(
-				&model.Universe{},
+
+			universe, err := model.NewUniverse(
+				model.WithProjectFrom("PROJECT"),
+				model.WithoutBoilerplate,
+				// No resource at vendor update
```

Comment:
I am not sure if I got the comment `// No resource at vendor update`.
Could you please clarify?

Reply:
There are three set of options to pass, for `Project`, `Boilerplate` & `Resource` fields. I'll clarify the comment (or maybe even delete it if it is confussing) with the next push.